src/components/OnboardingGuide/Guide/Guide.scss

```diff
@@ -0,0 +1,112 @@
+@use '../../variables';
+
+$block: '.#{variables.$ns}guide';
+
+$layer-popup: 1000;
+$header-height-s: 26px;
+$space-2-xs: 4px;
+$space-xs: 8px;
+$space-m: 16px;
+$space-s: 12px;
+$space-l: 20px;
+
+#{$block} {
+    z-index: $layer-popup;
+    max-width: 280px;
+
+    background: none;
+    width: fit-content;
+
+    &__content {
+        box-shadow: 0 2px 8px 0 rgba(0, 0, 0, 0.04);
```

Comment:
`rgba(0, 0, 0, 0.04)` not be visible in dark theme, you should use variable from sfx group